catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/SourceOperations.java

```diff
@@ -61,8 +63,12 @@
 
   private StorageProvider storage;
 
-  public SourceOperations(FrameworkProperties frameworkProperties) {
+  private SourceActionRegistry sourceActionRegistry;
+
+  public SourceOperations(
```

Comment:
Are the parameters guaranteed to not be `null`?

Reply:
No.

Reply:
So then what happens if they are null; and should there be protection against it?

Reply:
I added a `Validate.notNull(...)` for both args